index.html

```diff
@@ -14,19 +15,34 @@
     <!-- Navbar -->
     <div class="navbar">
       <a id="logo-text" href="#">Yuvenal</a>
-      <a class="nav-menu-icon" href="#"><img id="navbar-menu-icon" src="icons/menu/Union.png" alt="menu"></a>
+      <a class="nav-menu-icon" onclick="showMenu()"><img id="navbar-menu-icon" src="icons/menu/Union.png" alt="menu"></a>
       <ul class="navbar-menu">
-        <li><a class="nav-menu-button" href="#">Portfolio</a></li>
-        <li><a class="nav-menu-button" href="#">About</a></li>
-        <li><a class="nav-menu-button" href="#">Contact</a></li>
+        <li><a class="nav-menu-button" href="#works_section">Portfolio</a></li>
+        <li><a class="nav-menu-button" href="#about_section">About</a></li>
+        <li><a class="nav-menu-button" href="#contact_section">Contact</a></li>
         <li><a class="nav-menu-button" href="#">
             <img id="nav-mail-icon" src="images/desktop-size/Icon - Mail.png" alt="mail"></a>
         </li>
       </ul>
```

Comment:
![Figma](https://user-images.githubusercontent.com/93039359/201880342-e59961e1-06e4-455d-8d92-cdc8d8130d66.PNG)

- [ ] You are missing that section in your project menu which has been shown in the Figma image above 👍 Kindly add that section in your mobile menu to make it similar to the Figma template 😊